apps/agreement/contracts/Agreement.sol

```diff
@@ -383,7 +380,7 @@ contract Agreement is IAgreement, AragonApp {
         require(msg.sender == submitter, ERROR_SENDER_NOT_ALLOWED);
 
         IArbitrator arbitrator = _getArbitratorFor(action);
-        bytes memory metadata = _buildDisputeMetadata(action);
+        bytes memory metadata = abi.encodePacked(appId(), action.currentChallengeId);
```

Comment:
I do still like having a `:`, if only to make things a bit more obvious when looking at the raw bytes.

Would we be able to do this with `abi.encodePacked(appId(), ":", action.currentChallengeId)`?

Reply:
Oh nevermind, I see https://github.com/aragon/aragon-apps/pull/1189!

How much worse in gas is it to do avoid the `:`?

Reply:
I'll re-open the other one, I was inclined due to the gas costs, but actually I like the other one better. Let's check the gas costs diff and decide!

Reply:
Yes, that’s #1189

Reply:
Do you always remember by heart that `3a` corresponds to the colon? :astonished: